cmd/ocm/cluster/create/create.go

```diff
@@ -164,3 +161,26 @@ func run(cmd *cobra.Command, argv []string) error {
 
 	return nil
 }
+
+func fetchEnabledVersions(client *cmv1.RootClient) (versions []*cmv1.Version, err error) {
+	collection := client.Versions()
+	page := 1
+	size := 100
+	for {
+		var response *cmv1.VersionsListResponse
+		response, err = collection.List().
+			Search("enabled = 'true'").
+			Page(page).
+			Size(size).
+			Send()
+		if err != nil {
+			return
```

Comment:
stylistically, would it be better to not use naked returns?

Reply:
Well, I agree it is a matter of style/taste. I like them.